From 6b46cd26f9503ae50e9741747092a72d2fc35ec3 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Tue, 22 Feb 2022 05:44:57 +0100
Subject: [PATCH] build: add missing AM_CPPFLAGS to examples/

examples$ make V=1
depbase=`echo nft-buffer.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
gcc -DHAVE_CONFIG_H -I. -I..     -g -O2 -MT nft-buffer.o -MD -MP -MF $depbase.Tpo -c -o nft-buffer.o nft-buffer.c &&\
mv -f $depbase.Tpo $depbase.Po
nft-buffer.c:3:10: fatal error: nftables/libnftables.h: No such file or directory

Signed-off-by: Jan Engelhardt <jengelh@inai.de>
---
 examples/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/examples/Makefile.am b/examples/Makefile.am
index c972170d..d7234ce4 100644
--- a/examples/Makefile.am
+++ b/examples/Makefile.am
@@ -1,3 +1,4 @@
+AM_CPPFLAGS = -I$(top_srcdir)/include
 noinst_PROGRAMS	= nft-buffer		\
 		  nft-json-file
 
-- 
2.35.1

